src/panels/config/cloud/cloud-alexa-pref.ts

```diff
@@ -0,0 +1,83 @@
+import { html, LitElement, PropertyDeclarations } from "@polymer/lit-element";
+import { TemplateResult } from "lit-html";
+import "@polymer/paper-button/paper-button";
+import "@polymer/paper-card/paper-card";
+import "@polymer/paper-toggle-button/paper-toggle-button";
+// tslint:disable-next-line
+import { PaperToggleButtonElement } from "@polymer/paper-toggle-button/paper-toggle-button";
+
+import { fireEvent } from "../../../common/dom/fire_event";
+import { HomeAssistant } from "../../../types";
+import { updatePref } from "./data";
+import { CloudStatusLoggedIn } from "./types";
+
+export class CloudAlexaPref extends LitElement {
+  public hass?: HomeAssistant;
+  public cloudStatus?: CloudStatusLoggedIn;
+
+  static get properties(): PropertyDeclarations {
+    return {
+      hass: {},
+      cloudStatus: {},
+    };
+  }
+
+  protected render(): TemplateResult {
+    return html`
+      ${this.renderStyle()}
+      <paper-card heading="Alexa">
+        <paper-toggle-button
+          .checked='${this.cloudStatus!.alexa_enabled}'
```

Comment:
Should we be using `""` instead of `''` here? Just to keep inline with prettier. I don't think it sees those though